ibis/backends/base/sqlglot/__init__.py

```diff
@@ -52,11 +52,30 @@ def exists(self, query):
     def concat(self, *args):
         return sg.exp.Concat.from_arg_list(list(map(_to_sqlglot, args)))
 
+    def map(self, keys, values):
+        return sg.exp.Map(keys=keys, values=values)
+
+
+class ColGen:
+    __slots__ = ()
+
+    def __getattr__(self, name: str) -> sg.exp.Column:
+        return sg.column(name)
+
+    def __getitem__(self, key: str) -> sg.exp.Column:
+        return sg.column(key)
+
 
 def lit(val):
     return sg.exp.Literal(this=str(val), is_string=isinstance(val, str))
 
 
+def interval(value, *, unit):
+    return sg.exp.Interval(this=_to_sqlglot(value), unit=sg.exp.var(unit))
+
+
+f = FuncGen()
+c = ColGen()
```

Comment:
Single letter lowercase names intended to be imported by other modules feels weird to me. How about calling these `FUNCS` and `COLS` instead? Or `F` and `C` at least?

Reply:
`F` and `C` sounds good to me. Definitely want to keep them easy to type.

Reply:
Fixed!